src/cryptonote_core/loki_name_system_db.h

```diff
@@ -0,0 +1,113 @@
+#ifndef LOKI_NAME_SYSTEM_DB_H
+#define LOKI_NAME_SYSTEM_DB_H
+
+#include "crypto/crypto.h"
+#include "cryptonote_config.h"
+
+#include <string>
+
+struct sqlite3;
+struct sqlite3_stmt;
+namespace cryptonote
+{
+struct checkpoint_t;
+struct block;
+struct transaction;
+struct tx_extra_loki_name_system;
+struct account_address;
+}; // namespace cryptonote
+
+namespace lns
+{
+constexpr uint64_t BURN_REQUIREMENT                 = 100 * COIN;
+constexpr uint64_t BLOCKCHAIN_NAME_MAX              = 95;
+constexpr uint64_t BLOCKCHAIN_WALLET_ADDRESS_LENGTH = 69;
```

Comment:
Can you rewrite this value in terms of primitives?  i.e. `2*sizeof(public_key) + ...` or, if the header inclusions would be tricky, at least comment why the value is 69 instead of 64?  (A regular wallet address can be represented in 64, so what are the other 5 bytes doing?)